packages/common/src/types/sqlRunner.ts

```diff
@@ -100,13 +100,13 @@ export type BarChartSqlConfig = SqlRunnerChartConfig & {
 
 export type LineChartSqlConfig = SqlRunnerChartConfig & {
     type: ChartKind.LINE;
-    fieldConfig: SqlCartesianChartLayout | undefined;
+    fieldConfig: SqlPivotChartLayout | undefined; // PR NOTE: types are identical
     display: CartesianChartDisplay | undefined;
 };
 
 export type PieChartSqlConfig = SqlRunnerChartConfig & {
     type: ChartKind.PIE;
-    fieldConfig: SqlPieChartConfig | undefined;
+    fieldConfig: SqlPivotChartLayout | undefined; // PR NOTE: this will break serialization to the database (types are different)
```

Comment:
Pie chart `fieldConfig` is now identical to cartesian chart so it can use the same runner methods.

Is this too generic? Since `fieldConfig` supports pivots but the pie chart doesn't? This is my only uncertainty but I'm down to try this and see how it scales.